rest.insomnia.client.yml

```diff
@@ -0,0 +1,64 @@
+app-id: rest.insomnia.client
```

Comment:
It probably should be `rest.insomnia.Insomnia`, but it's up to you.

Reply:
Dunny why I went with that name, but your suggestion makes a lot more sense.